portfolio/src/main/webapp/searchClass.html

```diff
@@ -0,0 +1,225 @@
+                                <a class="nav-link drop" href="#">HOME</a>
+                            </li>
+
+                            <li class="nav-item">
+                                <a class="nav-link drop" href="#">CLASSES</a>
+                            </li>
+
+                            <li class="nav-item">
+                                <a class="nav-link drop" href="#">ABOUT</a>
+                            </li>
+
+                            <li class="nav-item">
+                                    <a class="nav-link drop" href="#">SIGN IN</a>
+                                </li>
+                        </ul>
+                    </div>
+                </div>
+            </nav>
+        </div>
+    </section>
+<!--Navigation Bar-->
+
+    <section class="search-form">
+        <!-- form header -->
+        <h1 class="form-header">Search For Class</h1>
+        
+        <!-- form contents -->
+        <form class="search-form-content" method="POST" action="/search-class">
+            <!-- row 1 -->
+            <div class="search-rows">
+                <input type="text" 
+                    class="text-input" 
+                    id="college-name" 
+                    name="schoolName"
+                    placeholder="College/University Name" required>
+                
+                <input type="text" 
+                    class="text-input" 
+                    id="department" 
+                    name="department"
+                    placeholder="Department" required>
+                
+                <input type="text" 
+                    class="text-input" 
+                    id="professor" 
+                    name="professor"
+                    placeholder="Professor">
+            </div>
+
+            <!-- row 2 -->
+            <div class="search-rows">
+                <input type="text" 
+                    class="text-input" 
+                    id="className" 
+                    name="className"
+                    placeholder="Class Name/Number" required>
+
+                <input type="text" 
+                    class="text-input" 
+                    id="classSemester" 
+                    name="classSemester"
+                    placeholder="Class Semester" required>
+            
+
+                <input type="text" 
+                    class="text-input" 
+                    id="classYear" 
+                    name="classYear"
+                    placeholder="Class Year" required>
+            
+                
+            </div>  
+            <button type="submit" class="btn">Search</button>  
+        </form>
+    </section>
+
+    <!-- pending: display the search result -->
+    <section class="result-table" >
+        <table>
+            <thead>
+                <th>College/University</th>
+                <th>Department</th>
+                <th>Class Info</th>
+                <th>Semester</th>
+                <th>Professor</th>
```

Comment:
The front-end looks amazing! Just one thing, can you add more space among the column headers? They look a bit condensed here.

![image](https://user-images.githubusercontent.com/89418223/178561752-282ed0ee-8a53-4f27-a73b-764caf6728d6.png)

Reply:
you are right! I was using a monitor so didnt count the scenario of laptops! I will fix it later!